docs/_src/tutorials/tutorials/5.md

```diff
@@ -289,7 +289,7 @@ More info on this metric can be found in our [paper](https://arxiv.org/abs/2108.
 
 ```python
 advanced_eval_result = pipeline.eval(
-    labels=eval_labels, params={"Retriever": {"top_k": 1}}, sas_model_name_or_path="cross-encoder/stsb-roberta-large"
```

Comment:
This change was automatically added to this PR because no automatic code and docs update were triggered by the already merged https://github.com/deepset-ai/haystack/pull/2550 to update `docs/_src/tutorials/tutorials/5.md`